code/controllers/subsystem/processing/quirks.dm

```diff
@@ -26,6 +26,7 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 		list("Clown Enjoyer","Mime Fan"),
 		list("Bad Touch", "Friendly"),
 		list("Extrovert", "Introvert"),
+		list("Nutriment Paste Fan", "Table Eating Enjoyer"),
```

Comment:
we are already HAVE the enjoyer/fan joke

Reply:
suggest a funnier joke then

Reply:
you dont need a JOKE